components/ide/jetbrains/gateway-plugin/gradle.properties

```diff
@@ -6,15 +6,15 @@ pluginName=gitpod-gateway
 pluginVersion=0.0.1
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=222.3048.1
+pluginSinceBuild=222.3325
 pluginUntilBuild=222.*
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions.
 pluginVerifierIdeVersions=2022.1, 2022.2
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#intellij-extension-type
 platformType=GW
 # Version from "com.jetbrains.gateway" which can be found at https://www.jetbrains.com/intellij-repository/snapshots
-platformVersion=222.3244.1-CUSTOM-SNAPSHOT
+platformVersion=222-NIGHTLY-CUSTOM-SNAPSHOT
```

Comment:
TODO: Update it to a fixed version when JetBrains team release a new Gateway Snapshot at https://www.jetbrains.com/intellij-repository/snapshots

The last version published is '222.3244.1-CUSTOM-SNAPSHOT', which doesn't have the `SshHostTunnelConnector` class.

After that, this PR will be ready for review.

Reply:
Updated.